converters/fromZigbee.js

```diff
@@ -3560,6 +3560,10 @@ const converters = {
                     result[postfixWithEndpointName('running_state', msg, model, meta)] = 'idle';
                 }
             }
+            if (msg.data.hasOwnProperty('occupiedHeatingSetpoint')) {
```

Comment:
This converter is also used by the Danfoss Icon thermostat, do you know if it also supports the `occupied_heating_setpoint_scheduled`, if yes then also add it to that device.

Reply:
@Koenkk  I separated the fromZigbee part (& squashed the last commits)